### DISCUSSION TITLE
What is your favorite animal?

### DISCUSSION THREAD

**Discussion by buccalcone** [NAIST-SE/AttractingContributors#1102] [2024 Attraction Contributions]
https://github.com/YU762/AttractingContributors-Animals

**Comment by Ninjia315** [comment]
hi, I submitted a request regarding Panda add-on in your list. 
If possible, could you mind helping me with my task?
https://github.com/Ninjia315/AttractingContributors-Favorite_Japanese_songs

**Reply by buccalcone** [reply]
Thank you for your help.
I added 和ぬか on your list.

**Comment by Canppchk** [comment]
I submitted a dog add-on in your list.
could you help me with my project too? Thanks!!
https://github.com/Canppchk/AttractingContributors-Ramen

**Comment by oyewaleoyebode** [comment]
I don't like pet

**Comment by Ngochuy2137** [comment]
I have just contributed to your repo. Can you contribute to my repo please https://github.com/Ngochuy2137/AttractingContributors-fruits/tree/main ?
